react-carousel-component/src/carousel.jsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+
+class Images extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = { count: 1 };
+    this.count = this.count.bind(this);
+    this.advance = this.advance.bind(this);
+    this.backwards = this.backwards.bind(this);
+    this.iconClick = this.iconClick.bind(this);
+
+  }
+
+  count() {
+    let increase = this.state.count;
+    if (increase < 5) {
```

Comment:
Try using the `length` of your passed in `props.poke` so you're not coupled to the size of your input

Reply:
🤔 actually, i don't think this `count` method is necessary anymore